lib/payoneer/payout.rb

```diff
@@ -2,14 +2,18 @@ module Payoneer
   class Payout
     CREATE_PAYOUT_API_METHOD_NAME = 'PerformPayoutPayment'
 
-    def self.create(program_id:, payment_id:, payee_id:, amount:, description:, payment_date: Time.now, currency: 'USD')
+    def self.create(program_id:, payment_id:, payee_id:, amount:, description:,
+                    payment_date: Time.current, currency: 'USD')
+      unless payment_date.is_a? String
+        payment_date = payment_date.in_time_zone('EST').strftime('%m/%d/%Y %H:%M:%S')
+      end
```

Comment:
I have document named Payoneer_integration_guide.pdf that states 
p9 (HTTP API)
Payment Date

[Required] Date of payment (EST), Normally, the
specified date is ignored, and the payment is executed
immediately upon payment submission.
A partner can be configured in the Payoneer system
for “Payments scheduling”. Using this configuration
(which is inactive by default), the payment is executed
only at the time specified by the “Payment date”
parameter.

Numeric
(MM/dd/yyyyHH:mm:ss)